lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -390,55 +391,6 @@ spec = do
                         (`shouldBe` reward)
                 ]
 
-    it "STAKE_POOLS_JOIN_01 - I can join another stake-pool after previously \
```

Comment:
removed as it is duplicate of what is in : 
```
it "STAKE_POOLS_JOIN/QUIT - Checking delegation expectations"
```
and we are close to timeout in CI